### DISCUSSION TITLE
What are the names of 🐪🐫?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1055] [Galaxy Trivia] [answered]
**Bonus:** Which one is critically endangered?

**Comment by peterbaccaro** [comment]
The **one-humped camel is commonly known as the dromedary camel**. Dromedaries (Camelus dromedarius) are domesticated camels found in many arid regions, particularly in the Middle East and North Africa. They are well-adapted to desert environments and have a single hump on their backs, which is used to store fat, not water as is commonly believed. 

The **two-humped camel is commonly known as the Bactrian camel**. Bactrian camels (Camelus bactrianus) are native to the Central Asian regions, including parts of China and Mongolia. They are distinct from the one-humped camels (dromedaries) and have two humps on their backs, which are used to store fat.

**Bactrian camels are critically endangered** in the wild due to hunting and competition with other livestock for food. There are fewer than 1,000 individuals remaining in their native range in the Gobi Desert in northern China and Mongolia – making them the eighth most endangered large mammal in the world.

**Comment by thuongtruong1009** [comment, marked as answer]
There are two main species of camels:

1. **Dromedary Camel (Camelus dromedarius)**: Also known as the Arabian Camel, this species is characterized by having a single hump on its back. Dromedary camels are native to the Arabian Peninsula and are also found in parts of Africa and South Asia. They are well adapted to desert environments and have been domesticated for thousands of years, playing a crucial role in the cultures and economies of the regions they inhabit.

2. **Bactrian Camel (Camelus bactrianus)**: The Bactrian Camel is native to Central Asia and is characterized by having two humps on its back. They are found in arid regions of Mongolia, China, Russia, Iran, Afghanistan, and other surrounding areas. Like the Dromedary Camel, Bactrian camels have also been domesticated for various purposes.

The critically endangered species of camel is the **Wild Bactrian Camel (Camelus ferus)**. This is a separate species from the domesticated Bactrian Camel (Camelus bactrianus). The Wild Bactrian Camel is native to the Gobi Desert region of China and Mongolia. Its population has been rapidly declining due to various factors, including habitat loss, climate change, and competition with domesticated camels. As of my last update in September 2021, it was estimated that there were only around 1,000 individuals left in the wild, making it one of the most endangered large mammal species in the world. Conservation efforts are ongoing to try to protect and preserve the remaining population of the Wild Bactrian Camel.

**Comment by Manishak798** [comment]
> **Bonus:** Which one is critically endangered?

Two humps camel.
